h/templates/5xx.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <title>Server Error</title>
+      <style>
```

Comment:
I sort of feel like that's a different issue altogether that's more appropriately solved in other ways, like pre-populating a CDN with new assets before deployment for instance.